docs/modules/ROOT/pages/vshn-managed/postgresql/create.adoc

```diff
@@ -38,6 +38,19 @@ spec:
 
 NOTE: To get more information about all available configuration options, please see the xref:references/crds.adoc#k8s-api-github-com-vshn-component-appcat-apis-vshn-v1-vshnpostgresql[API Reference]
 
+=== Sidecar Overhead and Fair-Use Policy
+
+Each PostgreSQL Standalone instance has following overhead:
+
+* CPU: 364m
+* Memory: 704Mi
+
+These are for various sidecar containers that run alongside the actual PostgreSQL instance.
```

Comment:
```suggestion
These are for various sidecar containers that run alongside the actual PostgreSQL instance.
```
The configuration applies to sidecar containers which run alongside PostgreSQL instance.

Reply:
I wouldn't call that configurations, as it's hardcoded and the user can't change them. Configurations sound like it's something the user could control.